navapbc/wic-mt-demo-project-eligibility-screener
# IncomeRow: Add data-label tags so column headers appear correctly on mobile

## Ticket

https://wicmtdp.atlassian.net/browse/WMDP-271

## Changes
> What was added, updated, or removed in this PR.

- IncomeRow: Add data-label tags so column headers appear correctly on mobile

## Context for reviewers
> Testing instructions, background context, more in-depth details of the implementation, and anything else you'd like to call out or ask reviewers. Explain how the changes were verified.

This bug was surfaced in [Tuesday's QA testing session](https://navasage.atlassian.net/wiki/spaces/MWDP/pages/513114154/UAT#%5Bdate%5D---Internal-QA-Testing).

## Testing
> Screenshots, GIF demos, code examples or output to help show the changes working as expected. ProTip: you can drag and drop or paste images into this textbox.

To test:
1. Build the image and run the container: `docker-compose -up -d --build`
1. Navigate to the screener at localhost:3000
1. Use the browser inspector to view on mobile or just shrink your browser to very skinny since the site is responsive
1. Go through the form wizard flow, make sure to check "none of the above" for adjunctive eligibility programs
1. Verify that the column headers are visible

Before with bug:
<img width="298" alt="Screen Shot 2022-11-03 at 3 00 46 PM" src="https://user-images.githubusercontent.com/67701/199842775-b7ae1c42-6d48-4fb6-b4e5-8f2d6376a1dd.png">

After with bug squashed:
<img width="302" alt="Screen Shot 2022-11-03 at 2 58 20 PM" src="https://user-images.githubusercontent.com/67701/199842817-1ec2b969-3b4d-4e6d-acd0-0d22984ab0a4.png">
